docs/modules/ROOT/pages/api/access.adoc

```diff
@@ -142,6 +145,194 @@ Emits an xref:OwnableComponent-OwnershipTransferred[OwnershipTransferred] event.
 
 Emitted when the ownership is transferred.
 
+[.contract]
+[[OwnableTwoStepComponent]]
```

Comment:
Following up the previous comment, this should not be a separated entry in here, but an update in the OwnableComponent entry.